code/game/objects/machinery/telecomms/ground_telecomms.dm

```diff
@@ -0,0 +1,146 @@
+#define TOWER_ON 0
+#define TOWER_OFF 1
+#define TOWER_BROKEN 2
+
+/obj/machinery/telecomms/relay/preset/tower
+	id = "Tower Relay"
+	autolinkers = list("relay")
+	icon = 'icons/obj/structures/comm_tower2.dmi'
+	icon_state = "comm_tower"
+	name = "TC-4T telecommunications tower"
+	desc = "A portable compact TC-4T telecommunications tower. Used to set up subspace communications lines between planetary and extra-planetary locations."
+	density = TRUE
+	use_power = NO_POWER_USE
+	idle_power_usage = 0
+	on = FALSE
+	freq_listening = TOWER_FREQS
+	resistance_flags = RESIST_ALL | DROPSHIP_IMMUNE
+	allow_pass_flags = PASS_PROJECTILE|PASS_AIR
+
+	///Health for the miner we use because changing obj_integrity is apparently bad
+	var/tower_integrity = 0
+	///Max health of the miner
+	var/max_tower_integrity = 100
+
```

Comment:
```suggestion
	///Health for the miner we use because changing obj_integrity is apparently bad
	var/tower_integrity = 0
	///Max health of the miner
	var/max_tower_integrity = 100
```